uptrain/operators/language/response_quality.py

```diff
@@ -870,16 +870,16 @@ def evaluate_local(self, data):
         Our methodology is based on the model grade evaluation introduced by openai evals.
         """
 
-        data_precision = copy.deepcopy(pl.DataFrame(data)).rename(
+        data_precision = copy.deepcopy(pl.DataFrame(data).drop('context')).rename(
```

Comment:
This will fail if sel.col_ground_truth = "context". We will need an alternative solution for this.